code/datums/supplypacks.dm

```diff
@@ -1177,6 +1177,16 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 	cost = 20
 	containername = "high-traction floor tiles"
 
+/datum/supply_packs/misc/plasmaman
+	name = "Plasma-man Supply Kit"
+	contains = list(/obj/item/clothing/under/plasmaman,
+					/obj/item/clothing/under/plasmaman,
+					/obj/item/weapon/tank/internals/plasmaman/belt/full,
+					/obj/item/weapon/tank/internals/plasmaman/belt/full,
+					/obj/item/clothing/head/helmet/space/hardsuit/plasmaman,
+					/obj/item/clothing/head/helmet/space/hardsuit/plasmaman)

```

Comment:
>10 point suits

A single suit is 80 points, and while that might be too much imo this is also kind of eh.